sources/packages/backend/libs/sims-db/src/entities/notification-metadata.type.ts

```diff
@@ -3,5 +3,7 @@
  */
 export interface NotificationMetadata {
   disbursementId?: number;
+  documentNumber?: number;
```

Comment:
Similar to a previous comment, is there a reason to add `documentNumber` and `errorCodes` to the metadata?